tests/src/core/testqgslabelingengine.cpp

```diff
@@ -4182,7 +4182,7 @@ void TestQgsLabelingEngine::labelingResultsWithCallouts()
   job.start();
   job.waitForFinished();
 
-  job.renderedImage().save( QStringLiteral( "/tmp/renderer.png" ) );
+  job.renderedImage().save( QString( "%s/renderer.png" ).arg( QDir::tempPath() ) );
```

Comment:
This isn't valid QString syntax. See https://doc.qt.io/qt-6/qstring.html#arg.

Reply:
Good catch! Handled in GH-57934